KenticoCloud.Delivery/ContentItemsInRichText/IInlineContentItemsResolver.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace KenticoCloud.Delivery.ContentItemsInRichText
+{
+    public interface IContentItemsInRichTextResolver
```

Comment:
please add comprehensible XML comments to all public members (especially interfaces)